service_mesh/v1x/ossm-traffic-manage.adoc

```diff
@@ -20,6 +22,6 @@ include::modules/ossm-routing-ingress.adoc[leveloffset=+1]
 
 include::modules/ossm-auto-route-1x.adoc[leveloffset=+1]
 
-== Links 
+== Links
```

Comment:
You could include an anchor ID for this heading. You could also consider making it an "Additional resources" heading, for consistency with other similar sections in the library. For more information, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies.